src/test/resources/expectedResultFromReport.csv

```diff
@@ -0,0 +1,6 @@
+"NewYork",101
+"NewYork",104
```

Comment:
why only 2 users of 4 are expected for "NewYork" location?

Reply:
Because only these IDs have frequency more than 10. So code picks any 5 or less entries which have been repeated more than 10 times.

Reply:
>  only these IDs have frequency more than 10.
what is fine, and I see it's being tested.


>  picks any 5 or less 

should also check "not more than 5" and "5"

Reply:
plus, it should pick  not "any", but **TOP**.